web/src/features/moreCast2/components/TabbedDataGrid.tsx

```diff
@@ -416,6 +413,8 @@ const TabbedDataGrid = ({ morecast2Rows, fromTo, setFromTo }: TabbedDataGridProp
     }
   }
 
+  const storedDraftForecast = new MorecastDraftForecast(localStorage)
```

Comment:
Might want to consider moving this into a local state variable (from the useState hook) as this instantiates a new MorecastDraftForecast object with evert render/re-render.

Reply:
Do you think it's necessary, since `MorecastDraftForecast` doesn't store anything within the class? It's kind of just getters and setters for data stored in localStorage. Just want to make sure I understand the reason